src/main/java/application/FormController.java

```diff
@@ -102,6 +111,7 @@ public SubmitResponseBean submitForm(@RequestBody SubmitRequestBean submitReques
         FormRecordProcessorHelper.processXML(formEntrySession.getSandbox(), formEntrySession.submitGetXml());
         SubmitResponseBean submitResponseBean = new SubmitResponseBean(formEntrySession);
         log.info("Submit response bean: " + submitResponseBean);
+        sessionRepo.delete(submitRequestBean.getSessionId());
```

Comment:
i can't either. though may want to just run a task that cleans up stale sessions. the only thing i could think that could wrong is if we think we've submitted it, but it has actually silently errored and we then delete the session